mesonbuild/compilers.py

```diff
@@ -2228,7 +2232,7 @@ def get_compiler_check_args(self):
         # too strict without this and always fails.
         return self.get_no_optimization_args() + ['-fpermissive']
 
-class GnuObjCCompiler(GnuCompiler,ObjCCompiler):
+class GnuObjCCompiler(GnuCompiler, ObjCCompiler):
```

Comment:
You really shouldn't touch code unnecessarily; you'll get code style warnings about stuff that you really don't need to be fixing in this PR.